### PR TITLE
Update configuratiot settings to use new table format

### PR DESCRIPTION
**Pull request by johnsimons**

### CODE REVIEW

File: servicecontrol/servicecontrol-instances/configuration.md [hunk 1]
```diff
@@ -532,39 +570,74 @@ Version: 5.4.0+
...
 
-Type: string
+| Context | Name |
+| --- | --- |
+| **Environment variable** | `LICENSECOMPONENTS_AMAZONSQS_ACCESSKEY` |
+| **App config key** | `LicensingComponent/AmazonSQS/AccessKey` |
+
+| Type | Default value |
+| --- | --- |
+| string | existing transport connection string |
```
Comment (jpalac) [inline servicecontrol/servicecontrol-instances/configuration.md:580]: I would change this text to be a bit more specific, perhaps:
```suggestion
| string | obtained from ServiceControl transport connection string |
```

**Review by jpalac — COMMENTED** [review]
Can you do the same to the monitoring instance setting http://localhost:55666/servicecontrol/monitoring-instances/configuration#usage-reporting

File: servicecontrol/servicecontrol-instances/configuration.md (hunk 1 above)
Comment (johnsimons) [reply servicecontrol/servicecontrol-instances/configuration.md]: fixed

**Review by jpalac — APPROVED** [review]

File: servicecontrol/servicecontrol-instances/configuration.md [hunk 2]
```diff
@@ -463,9 +463,14 @@ If running multiple setups of the Platform Tools (i.e. multiple versions of Serv
 
 If using [MSMQ transport](/transports/msmq) and the monitoring instance is installed on a different machine than the ServiceControl error instance, only the monitoring instance setting needs to be modified to include the machine name of the error instance in the queue address.
 
-Type: string
+| Context | Name |
+| --- | --- |
+| **Environment variable** | `LICENSECOMPONENTS_SERVICECONTROLTHROUGHPUTDADAQUEUE` |
```
Comment (abparticular) [inline servicecontrol/servicecontrol-instances/configuration.md:468]:
```suggestion
| **Environment variable** | `LICENSECOMPONENTS_SERVICECONTROLTHROUGHPUTDATAQUEUE` |
```

Comment (abparticular) [reply servicecontrol/servicecontrol-instances/configuration.md:468]: Also is it correct that app config prefix is `LicensingComponent` and env var prefix is `LICENSECOMPONENTS` ?

Comment (jpalac) [reply servicecontrol/servicecontrol-instances/configuration.md:468]: I'd say one of those is wrong - the answer to which one would be in code.

Comment (johnsimons) [reply servicecontrol/servicecontrol-instances/configuration.md:468]: that may be a mistake on my side

**Pull request merged**